scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -83,9 +84,8 @@ emcmake cmake \
 make -j 4
 
 cd ..
-cp build/solc/soljson.js ./
 mkdir -p upload
-cp soljson.js upload/
```

Comment:
Ah now I see the full context. I think soljson in the root directory is also required by some test scripts.